Jonathan/Week_6/stations_challenge.py

```diff
@@ -0,0 +1,125 @@
+'''
+NYC Transit Challenge! 
+
+In this challenge, you will use OOP and inheritance to design subway and bus stations!
+You'll use the parent class Station to make child classes for SubwayStation and BusStation. 
+
+Note, you should NOT need to change the Station class at all in this challenge.
+
+Since subways and buses have different information, the methods and attributes will be a bit different
+'''
+
+
+print('Question 1: Making the SubwayStation Class')
+'''
+Using the Station class below as the parent, make a child class called SubwayStation
+
+SubwayStation should:
+-have an additional attribute called 'lines' that is user-defined as a list during initialization. 
+    this will indicate which subway lines stop at the station (for example ['A', 'C'])
+-override the show_info() method from Station to display which subway lines stop there, in addition to the station_name and location
+'''
+class Station:
+    def __init__(self, station_name, location):
+        self.station_name = station_name
+        self.location = location
+    
+    def show_info(self):
+        print(f'{self.station_name} station is located at {self.location}')
+
+class SubwayStation(Station):
+    def __init__(self, station_name, location, lines):
+        super().__init__(station_name, location)
+        self.lines = lines
+    
+    def show_info(self):
+        print(f'{self.station_name} station is located at {self.location}, and the line {self.lines} stop here.')
+
+print()
+print('Question 2: Make an example subway station')
+'''
+Using your SubwayStation class, instantiate a subway station with the info below. 
+Then run the show_info() method to make sure you get the station_name, location, and lines printed out
+
+station_name: '14th street'
+location: '14th street and 7th avenue'
+lines: ['1', '2', '3', 'L']
+'''
+
+station_union_square = SubwayStation('14th street', '14th street and 7th avenue', ['1', '2', '3', 'L'] )
+
+station_union_square.show_info()
+print()
+
+print('Question 3: Making the BusStation Class')
+
+'''
+Using the Station class below as the parent, make a child class called BusStation
+
+BusStation should:
+-have an additional attribute called 'routes' that is user-defined as a list during initialization. 
+    this will indicate where buses can go from this station. For example, ['DC', 'Philly', 'Pittsburgh']
+-have an additional attribute called 'open' that is always initalized as True (a boolean variable)
+-have additional methods called open_station() and close_station() to open and close the station
```

Comment:
hey Jon! this is a great start--however, you forgot to add the open_station() and close_station() methods. You also need to test these methods work in Q5.

ex.
open_station() should set self.open == True, while close_station should set self.open == False. 
The work in the open_station() method right now should be put inside the show_info() method, which we need to override in this example. Please consider OH if this doesn't make sense, especially because it's new info